### PR TITLE
added action.yaml and scripts to install latest

### PR DESCRIPTION
**Pull request by garethahealy**
adds:
- github action to test the prepackaged on PR
- github action to build/release to ghrc.io both lazy and prepackaged
- upgrades the prepackaged to latest LTS as base image is out of date (happy to move this out if the base can be upgraded/released - I had tons of issues using the base jenkins due to plugin versions etc)
- adds action.yaml so can provide input

### CODE REVIEW

**Review by jonico — APPROVED** [review]
Looks great to me

**Comment by jonico** [comment]
Looks good to me, but would ❤️ a second pair of 👀 from @oleg-nenashev before merging

**Review by oleg-nenashev — APPROVED** [review]
It is definitely improvement compared to the previous state, +1 for merging

Later I suggest replacing the Docker image and plugin sets by the Vanilla Image of JFR

File: jenkinsfile-runner-prepackaged/Dockerfile.build
Comment (oleg-nenashev) [inline jenkinsfile-runner-prepackaged/Dockerfile.build:19]: Yeah, my bad Too many emergencies in personal life, co-maintainers are always welcome

File: jenkinsfile-runner-prepackaged/Dockerfile.build
Comment (oleg-nenashev) [inline jenkinsfile-runner-prepackaged/Dockerfile.build:1]: FTR https://github.com/jenkinsci/jenkinsfile-runner/pkgs/container/jenkinsfile-runner can be used for the latest version of Jenkinsfile Runner

**Comment by oleg-nenashev** [comment]
FTR I started https://github.com/jenkinsci/jenkinsfile-runner-github-actions/pull/14 @garethahealy @jonico

**Pull request merged**
